### PR TITLE
add a counter

### PR DESCRIPTION
**Pull request by kimberleykelly**
Screen Shot 2019-06-24 at 17 13 30

### CODE REVIEW

**Comment by gmcquistin** [comment]
Great job 🎉

File: src/counter.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      start: 0
```
Comment (neil-morrison44) [inline src/counter.js]: why `start`?

Comment (kimberleykelly) [reply src/counter.js]: because the counter has a starts at 0?

Comment (neil-morrison44) [reply src/counter.js]: but it doesn't stay at 0 & you're still calling it `start`

**Pull request merged**